src/layouts/default.vue

```diff
@@ -80,20 +76,36 @@ const embeddedPage = {
         isSearchRoute.value && !isHeaderScrolled.value && !isMinScreenMd.value
     )
     provide('headerHasTwoRows', headerHasTwoRows)
+
```

Comment:
We should probably remove this CSS variable if we removed the scrolled/non-scrolled padding change.